README.md

```diff
@@ -30,18 +38,18 @@ interface Foo
 }
 ```
 
-5. Use inline PHPDoc only for fields inside the class, like:
+8. Use **inline** PHPDoc only for fields inside the class and method return type (if not specified by the code):
 
 ```php
 final class Foo
 {
     /** @var int */
     private $foo;
     
-    /** @var string */
-    private $bar;
+    private string $bar;
     
-    public function getFoo(): ?int
+    /** @return int|null */
```

Comment:
why here comment instead of '?int'

Reply:
Just for example - if you use PHP, which doesn't support property types, that's valid line.